src/lib/utils/application-commands/ApplicationCommandRegistry.ts

```diff
@@ -371,7 +398,9 @@ export class ApplicationCommandRegistry {
 			);
 		}
 
-		const header = `Found differences for command "${applicationCommand.name}" (${applicationCommand.id}) versus provided api data\n`;
+		const header = `Found differences for command "${applicationCommand.name}" (${applicationCommand.id}) versus provided api data.${
+			finalMessage.length ? '\n' : ''
+		}`;
```

Comment:
```suggestion
		const finalMessageNewLine = finalMessage.length ? '\n' : '';
		const header = `Found differences for command "${applicationCommand.name}" (${applicationCommand.id}) versus provided api data.${finalMessageNewLine}`;
```

This will make the code a bit more readable